### PR TITLE
ARD: 0011. Tweak PR process [no code]

### PR DESCRIPTION
**Pull request by oskarth**
ARD from Core Dev call re PR process.

### CODE REVIEW

File: doc/decisions/0011-tweak-pr-process.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+These specific policy changes were agreed upon (a la 'rough consensus' model). These are heuristics, and should be more of a guide than strict rules. Exception to these should be minimal though.
+
+**1. Reject PRs that don't have tests**
+As a rule of thumb, coverage should increase.
+
+**2. Don't use or promote nightly as something for end users.**
+
+**3. Reject PRs that don't write what tests the author have done and what platforms you used to test, including screenshots.**
+This means QA and dev have to spend less time catching obvious problems.
```
Comment (mandrigin) [inline doc/decisions/0011-tweak-pr-process.md:30]: "including screenshots, videos or logs" I would say

**Review by mandrigin — APPROVED** [review]
a tiny note, but lgtm

**Comment by oskarth** [comment]
@annadanchenko @hesterbruikman Ping, please try to review within a work day so we can speed up integration.

**Review by annadanchenko — APPROVED** [review]

File: doc/decisions/0011-tweak-pr-process.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+**4. Run tests when a PR is at "Review" stage on https://github.com/status-im/status-react/projects/7 (or add new 'e2e stage' between review and `TO TEST`)**
+
+**5. Dev asks for 2-3 reviewers and these reviewers review within a day.**
```
Comment (hesterbruikman) [inline doc/decisions/0011-tweak-pr-process.md]: I would extend: "2-3 reviewers, including a designer if the change affects UI"

**Comment by hesterbruikman** [comment]
Added one comment, other than that looks good.

**Comment by oskarth** [comment]
Thanks, fixed

**Pull request merged**